orm/models.py

```diff
@@ -95,13 +95,21 @@ def build_select_expression(self):
         return expr
 
     def filter(self, **kwargs):
+        return self._parse_filters(**kwargs)
+
+    def exclude(self, **kwargs):
+        return self._parse_filters(exclude=True, **kwargs)
+
+    def _parse_filters(self, exclude=False, **kwargs):
```

Comment:
@EmilioCarrion, even with your latest changes, `exclude` could conflict one of the parameters used for filtering. Take a look at the following example:

```python
class Person(orm.Model):
    id = orm.Integer(primary_key=True)
    exclude = orm.String(max_length=100)

await Person.filter(exclude=True).all()
```

The column name `exclude` conflicts with the `exclude` parameter in `_parse_filters`.

One solution would be to change the definition of `_parse_filters` to the following:

```python
def _parse_filters(self, values, exclude=False):
```

Alternatively, you could rename `exclude` to `_exclude`.